runtime/src/nfts.rs

```diff
@@ -158,10 +159,15 @@ mod tests {
 		pub const TestChainId: u8 = 5;
 	}
 
+    ord_parameter_types! {
+		pub const One: u64 = 1;
+	}
+
 	impl chainbridge::Trait for Test {
 		type Event = ();
 		type Proposal = Call;
 		type ChainId = TestChainId;
+        type AdminOrigin = EnsureSignedBy<One, u64>;
```

Comment:
Indentation is somehow broken?

Reply:
I think is the classic tale of tabs vs spaces. My IDE transforms tabs in spaces, but I believe the initial code was with tabs. It looks good on an editor anyways.